### ISSUE TITLE
No intellisense when hovering over keyword argument in function call

### ISSUE THREAD

**Issue by mjbvz** [microsoft/python-language-server#1970]
## Environment data

- VS Code version:  1.44.0-insider
- Extension version (available under the Extensions sidebar): 2020.4.68586-dev
- OS and version: MacOS 10.15.3 
- Python version (& distribution if applicable, e.g. Anaconda): 3.7.3
- Type of virtual environment used (N/A | venv | virtualenv | conda | ...): NA
- Relevant/affected Python packages and their versions: NA
- Relevant/affected Python-related VS Code extensions and their versions: NA
- Jedi or Language Server? (i.e. what is `"python.jediEnabled"` set to; more info microsoft/vscode-python#3977): Language server
- Value of the `python.languageServer` setting: Microsoft

## Steps to reproduce:
1. For the python file:

```py
({'x': 1}).pop('x', d=)
```

2. Hover over `d`

## Expected behaviour
IntelliSense for this keyword argument is shown

## Actual behaviour
No intellisense

Other example:

```py
import argparse

argparse.ArgumentParser(description="") # hover over 'description'
```

**Comment by MikhailArkhipov** [comment]
What is expected from the hover? Hovering over parameter may show type, but it depends. Are you expecting some documentation on the argument? I.e. fragment of the class docstring like
```
 - description -- A description of what the program does
```
?

**Comment by mjbvz** [comment]
Yes, I'd expect any documentation associated with the parameter to be displayed

**Comment by jakebailey** [comment]
Note that Python does not have any official specification for this (there are at least 3 popular forms I know off the top of my head). We currently don't do docstring "parsing", just a loose conversion from docstrings to valid markdown to make them better than just VS Code's plaintext mode (which wraps poorly).

Docstrings written in the standard library in particular are only written to be visible via the `help()` builtin function in a terminal, and conform to no markup language. Argparse's is unfortunately not one of the more well-known formats (sphinx/numpy/google/etc).
